jans-cedarling/schema/cedarling_core.cedarschema

```diff
@@ -90,42 +90,42 @@ namespace Jans {
 
     // ******  Actions  ******
     action Compare appliesTo {
-      principal: [User, Workload],
+      principal: [User, Workload, Role],
```

Comment:
Why you add `Role` as principal? 
For checking role we use hierarchy membership `cedar-policy`
https://docs.cedarpolicy.com/policies/syntax-operators.html#operator-in

Reply:
Because Agama Lab cannot create policy like 
```
permit(
principal in Role,
...
)
```
if schema does not have any actions with Role as principal

Reply:
@SafinWasi if it is hack for demo, it is OK. But agama lab should be updated to support it. Because it is wrong implementation

Reply:
Ok, for now let's merge it, I will revert it once Agama Lab has fixed this issue